### DISCUSSION TITLE
Invalid REACH_CONNECTOR_MODE: ALGOexport

### DISCUSSION THREAD

**Discussion by DanBurton** [reach-sh/reach-lang#1029] [Q&A] [answered]
Discord user 3dlife reported:

> Reset Docker and made a new install and error persist with all my reach apps. Error when try to run on terminal ./reach compile ---- Invalid REACH_CONNECTOR_MODE: ALGOexport

Discord user onyedikachi also reported the same issue:

> I'm getting this error: Invalid REACH_CONNECTOR_MODE: ALGOexport after updating reach this morning.

**Comment by DanBurton** [comment, marked as answer]
Both users reported that manually editing the reach config file was sufficient to address the issue.

onyedikachi said:

> Resolved: Edited the env file in Home/config/reach

3dlife said:

> Remove the dot from the folder named  ".config" That's it. just copy the root path from docker Mount and search for the config path.

(I -- Dan -- only see the file at `~/.config/reach/env` on my machine.)

**Reply by 3dlifee** [reply]
That's it. I only remove the dot and it works
<img width="341" alt="Screenshot 2022-05-21 at 19 11 18" src="https://user-images.githubusercontent.com/74255371/169670749-e6ebf6eb-4070-4f46-8191-3d7a7362032a.png">
.
